lib/y2_logs_helper.pm

```diff
@@ -101,14 +101,22 @@ sub accept_license {
 }
 
 sub verify_license_translations {
-    return if (is_sle && get_var("BETA") || check_var('VIDEOMODE', 'text'));
+    return if (is_sle && get_var("BETA"));
     my $current_lang = 'english-us';
     for my $lang (split(/,/, get_var('EULA_LANGUAGES')), 'english-us') {
         wait_screen_change { send_key 'alt-l' };
-        assert_and_click "license-language-selected-$current_lang";
-        wait_screen_change { type_string(substr($lang, 0, 1)) };
+        # in textmode only arrow navigation is possible
+        if (get_var('VIDEOMODE') =~ 'text') {
+            send_key_until_needlematch("license-language-selected-english-us", 'up', 60);
+            send_key 'ret';
+        }
+        else {
+            assert_and_click "license-language-selected-$current_lang";
+        }
+        
+        wait_screen_change { type_string(substr($lang, 0, 1)) } if (!get_var('VIDEOMODE')); # if not in textmode
```

Comment:
you can be explicit here, not need for the comment: ` If !check_var('VIDEOMODE', 'textmode');`

Reply:
what do you mean explicit?

Reply:
VIDEOMODE has more values and you just writing in the comment what you would express just with code.

Reply:
e308b008a